src/components/app/details/cicdHistory/TriggerDetails.tsx

```diff
@@ -0,0 +1,295 @@
+import { cancelCiTrigger, cancelPrePostCdTrigger } from '../../service'
+
+export const TriggerDetails = React.memo(
+    ({
+        status,
+        startedOn,
+        finishedOn,
+        triggeredBy,
+        triggeredByEmail,
+        ciMaterials,
+        gitTriggers,
+        message,
+        podStatus,
+        type,
+        stage,
+        artifact,
+    }: TriggerDetailsType) => {
+        return (
+            <div
+                className="trigger-details"
+                style={{ height: '137px', display: 'grid', gridTemplateColumns: '60px 1fr' }}
+            >
+                <div className="trigger-details__status flex">
+                    <svg width="25" height="87" viewBox="0 0 25 87" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <circle cx="12.5" cy="6.5" r="6" fill="white" stroke="#3B444C" />
+                        <circle
+                            cx="12.5"
+                            cy="74.5"
+                            r="6"
+                            fill={colorMap[status?.toLowerCase()]}
+                            stroke={colorMap[status?.toLowerCase()]}
+                            strokeWidth="12"
+                            strokeOpacity="0.3"
+                        />
+                        <line x1="12.5" y1="11.9997" x2="12.5362" y2="69" stroke="#3B444C" />
+                    </svg>
+                </div>
+                <div className="trigger-details__summary" style={{ display: 'grid', gridTemplateRows: '1fr 1fr' }}>
+                    <StartDetails
+                        startedOn={startedOn}
+                        triggeredBy={triggeredBy}
+                        triggeredByEmail={triggeredByEmail}
+                        ciMaterials={ciMaterials}
+                        gitTriggers={gitTriggers}
+                        artifact={artifact}
+                        type={type}
+                    />
+                    <CurrentStatus
+                        status={status}
+                        finishedOn={finishedOn}
+                        artifact={artifact}
+                        message={message}
+                        podStatus={podStatus}
+                        stage={stage}
+                        type={type}
+                    />
+                </div>
+            </div>
+        )
+    },
+)
+
+const Finished = React.memo(
+    ({ status, finishedOn, artifact }: FinishedType) => {
+        return (
+            <div className="flex column left">
+                <div
+                    className={`${status} fs-14 fw-6 ${
+                        TERMINAL_STATUS_COLOR_CLASS_MAP[status.toLowerCase()] || 'cn-5'
+                    }`}
+                >
+                    {status && status.toLowerCase() === 'cancelled' ? 'ABORTED' : status}
+                </div>
+                <div className="flex left">
+                    {finishedOn && finishedOn !== '0001-01-01T00:00:00Z' && (
```

Comment:
keep this in const `'0001-01-01T00:00:00Z'` or use existing one if present

Reply:
Done